src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -152,11 +156,13 @@ public static class SnackbarVisualElementExtension
 	/// <param name="action">Action of the snackbar button</param>
 	/// <param name="duration">Snackbar duration</param>
 	/// <param name="visualOptions">Snackbar visual options</param>
+	/// <param name="token">Cancellation token</param>
 	public static Task DisplaySnackbar(
 		this VisualElement? visualElement,
 		string message,
 		Action? action = null,
 		string actionButtonText = "OK",
 		TimeSpan? duration = null,
-		SnackbarOptions? visualOptions = null) => Snackbar.Make(message, action, actionButtonText, duration, visualOptions, visualElement).Show();
+		SnackbarOptions? visualOptions = null,
+		CancellationToken token = default) => Snackbar.Make(message, action, actionButtonText, duration, visualOptions, visualElement).Show(token);
```

Comment:
Lazy question - What is the `default` value for `CancellationToken`? Is it `CancellationToken.None`?

Reply:
yes,
![image](https://user-images.githubusercontent.com/33021114/148657113-223bad0f-7254-453f-9942-d2d764d6025e.png)

we cannot use CancellationToken.None because the default parameter must be a compile-time constant